rapid7_insightvm/komand_rapid7_insightvm/util/resource_requests.py

```diff
@@ -0,0 +1,202 @@
+from .shared_resources import *
+import json
+import requests
+import urllib3
+from komand.exceptions import PluginException
+from typing import NamedTuple, Collection
+
+# Suppress insecure request messages
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+class RequestResult(NamedTuple):
+    """
+    Class for data returned in paged InsightVM API requests
+    This is the Python 3.6 style for using named tuples with typing
+    """
+    page_num: int
+    total_pages: int
+    resources: dict
+
+
+class TestResult(NamedTuple):
+    """
+    Class for data returned in test method
+    This is the Python 3.6 style for using named tuples with typing
+    """
+    status: int
+    message: str
+
+
+class ResourceRequests(object):
+    """
+    Class for helper methods for making requests against the APIv3. A new instance should
+    be instantiated within the action/trigger being developed. New methods should be
+    created as instance methods to allow reference of the logger and session passed to
+    the __init__ function during instantiation.
+    """
+
+    # Static headers for all requests
+    _HEADERS = {
+        'Content-Type': 'application/json',
+        'Accept': 'application/json'
+    }
+    _ENSURECONNECTIVITY = 'Ensure proper network connectivity between the InsightConnect orchestrator and the InsightVM console'
+
+    # Currently only handling the most common requests exceptions more can be added as needed
+    _REQUEST_EXCEPTIONS = {
+        requests.HTTPError: 'If this issue persists contact support for assistance.',
+        requests.ConnectionError: 'Unable to connect to IVM console.'
+                                  'If this issue persists contact support for assistance.',
+        requests.Timeout: _ENSURECONNECTIVITY,
+        requests.ConnectTimeout: _ENSURECONNECTIVITY,
+        requests.ReadTimeout: _ENSURECONNECTIVITY,
```

Comment:
Verify these two constants  (ConnectTimeout and ReadTimeout) exist. PyCharm cannot find them.